src/Components/Results.js

```diff
@@ -3,14 +3,16 @@ import React from 'react';
 const Results = (props) => {
 
     const { repos } = props;
-    //console.log('repos is ', repos.data);
+   
 
     const listRepos = repos.length !==0 ?
         repos.data.map((item) => <li key={item.id}> {item.name} </li>) : (
-          <br />);
+            <br />);
```

Comment:
Using br is not a good practice, especially if it stands as a separate element as it's here. Maybe you can add a div with info 'No repos'.